src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -989,6 +990,11 @@ const Authenticate = (() => {
     };
 
     const onLoad = async () => {
+        if (!ClientBase.isAuthenticationAllowed()) {
+            $('#authentication_tab').setVisibility(0);
+            $('#authentication_loading').setVisibility(0);
+            $('#error_occured').setVisibility(1);
```

Comment:
this shows the error `there was a connection error. Please try again later.` which is wrong. we should show the same text as the pop up, that client does not need to authenticate.